src/MagnumPlugins/WavAudioImporter/Test/WavImporterTest.cpp

```diff
@@ -172,6 +172,10 @@ void WavImporterTest::invalidFactChunk() {
 
     CORRADE_COMPARE(importer.format(), Buffer::Format::Mono16);
     CORRADE_COMPARE(importer.frequency(), 22050);
+
+    CORRADE_COMPARE_AS(importer.data().slice(0, 8),
+        Containers::Array<char>::from(-27, -11, -1, -9, 24, -6, 127, -5).slice(0, 8),
```

Comment:
@mosra That is, frankly, substantially longer than say:

    Containers::Array<char>::from(-27, -11, -1, -9, 24, -6, 127, -5).prefix(8)

Which also looks much more like:

    importer.data().prefix(8)